### ISSUE TITLE
The user should be able to zoom in and out of the preview.

### ISSUE THREAD

**Issue by fabiocolacio** [fabiocolacio/Marker#94]
This can be achieved with ``webkit_web_view_set_zoom_level ()``. This setting should probably be persistent across sessions with GSettings also.

**Comment by Mandarancio** [comment]
commit bc2be32 solves the issue:

|Short cut| Action |
|:----|:----|
|```ctrl + scroll```|zoom in/out |
|```ctrl + +```|zoom in |
|```ctrl + -```|zoom out |
|```ctrl + 0```|reset zoom|

I suggest however to add a +/- button and label whit current zoom level in the menu
